### ISSUE TITLE
Not working under Qutebrowser

### ISSUE THREAD

**Issue by Jaegrqualm** [rcc11/4chan-sounds-player#3]
As the title says. Doesn't show up at all. Other loaded scripts: textarea-backup, 4chan-x, 4chan External Sounds (via greasyfork, some time ago)

 Some previous version has worked(also via greasyfork), albeit it was still unstable and eventually stopped working altogether, after removing all of the `@connect` and `@grant` stuff from the header. It could've been that it was merely just unstable the whole time.
Sorry I can't be more specific, the simple version has worked for me just fine so I hadn't considered filing an issue.

Error message in the console:
```
userscript:GM-4chan sounds player:4386 Uncaught TypeError: Cannot read property 'classList' of null
    at Object../src/globals.js (userscript:GM-4chan sounds player:4386)
    at __webpack_require__ (userscript:GM-4chan sounds player:229)
    at Module../src/main.js (userscript:GM-4chan sounds player:4503)
    at __webpack_require__ (userscript:GM-4chan sounds player:229)
    at userscript:GM-4chan sounds player:293
    at userscript:GM-4chan sounds player:296
    at userscript:GM-4chan sounds player:5876
```

**Comment by Jaegrqualm** [comment]
relevant reading, https://github.com/qutebrowser/qutebrowser/issues/3238
It appears  that qutebrowser's greasemonkey support may not be fully up to snuff, and that may be what's causing the issue. Looking at that list myself, I couldn't name anything in particular as being the cause, though.

**Comment by Jaegrqualm** [comment]
Ahhh, well, [this comment](https://github.com/qutebrowser/qutebrowser/issues/3238#issuecomment-423851046) gave me a hint that changing the `document-start` to `document-end` may help, and well, it's working. full-fat 3.1.2 release with only that one line changed. If @rcc11 doesn't see any problem with that change, it would be great. Waiting on a response before closing.  I could issue a pull request, but i mean, it's one line.

**Comment by rcc11** [comment]
@Jaegrqualm The problem is setting run-at to document-end breaks the 4chan X detection and anything the depends on it. I'll do some investigation to see if I can get it working with document-end but that detection has always been a particular pain to keep working in all cases. It has a habit of causes no sounds to be loaded if everything isn't timed up right.

Dropping the error image here so I don't have to look it up in the archive. A different fix might be to not reference documentElement where this error is occurring, or hold off on referencing it.
![image](https://user-images.githubusercontent.com/64926786/96348240-40ec3b00-109f-11eb-803a-e70540b9cecf.png)

**Comment by rcc11** [comment]
@Jaegrqualm I think just adding [this check](https://github.com/rcc11/4chan-sounds-player/commit/b73600d2fd7fe4635912e518420e4896be26937b) for documentElement fixes it. It won't have any effect elsewhere where it's defined, and I wasn't seeing any issues in the 4chan X detection with it.

It's included in the beta if you want to give it a try and see if you're getting any issues still. If you're running 4chan X then you should see "4chan X Header Controls" in the display settings under "Minimised Display". That's the easiest way to check the detection.

**Comment by Jaegrqualm** [comment]
@rcc11 After a quick test, that appears to be working. 
Quick edit: Ahhh, but there may be problems elsewhere. Going to test further.

**Comment by Jaegrqualm** [comment]
Yeah, so at the very least the "jump to post for the current sound" button isn't working with the beta compared to my changed version of 3.1.2. Hard for me to say beyond that, but at least the beta loads.

**Comment by rcc11** [comment]
@Jaegrqualm That's an unrelated bug introduced with some other changes. https://github.com/rcc11/4chan-sounds-player/commit/324257d41e840cd5d26bbaa5dbc4746d6da7e3d7

**Comment by Jaegrqualm** [comment]
@rcc11 Ah, well then. I'll check back in a bit.
